packages/react-ui-ag/src/Gmap/__tests__/Markers-test.js

```diff
@@ -1,30 +1,49 @@
 import React from 'react'
 import { shallow } from 'enzyme'
 import Markers from '../Markers'
-import { markerRedDotIcon, markerBlackDotIcon } from '../markerIcons'
+import { redDotIcon, greyDotIcon, blackDotIcon, blackDotIconWithBalloon } from '../markerIcons'
 
 const map = { google: 'map' }
 
+const NOOP = () => {}
+
 describe('ag/Markers', () => {
   it('applies extra props correctly', () => {
     const wrapper = shallow(<Markers map={map} data-tag_section="marker-tag" />)
     expect(wrapper.prop('data-tag_section')).toEqual('marker-tag')
   })
 
   describe('marker', () => {
-    it('uses default marker props when none passed', () => {
+    it('uses a redDotIcon and default marker props when none passed', () => {
       const wrapper = shallow(<Markers map={map} />)
       expect(wrapper.instance().marker()).toEqual({
-        icon: markerRedDotIcon(),
+        icon: redDotIcon(),
         onMouseOver: expect.any(Function),
         onMouseOut: expect.any(Function),
       })
     })
+    it('uses a greyDotIcon for inactive properties', () => {
+      const markers = new Markers()
```

Comment:
How can you pass `feature` if you shallow render `Markers`? It should be sufficient to invoke the getter from the class instance and assert that the logic is correct.

Reply:
I mean I guess you could to is this way. The normal way would be something like this:
```
const wrapper = shallow(<Markers map={map} markerIconHover={NOOP} />)
const instance = wrapper.instance()
const icon = greyDotIcon()

expect(instance.marker(feature)). toContain({ icon })
```
(haven't tested this, but something like this would work)

This will actually matter if you make a default `markertIconInactive` prop.